### DISCUSSION TITLE
cef的wheel, mouseDown无法监听到mouseEvent的CTRL KEY按下

### DISCUSSION THREAD

**Discussion by L-ZT** [CefView/QCefView#218] [Q&A]
在谷歌浏览器是正常的

**Comment by L-ZT** [comment]
求助求助！

**Reply by tishion** [reply]
什么信息都不提供，别人怎么帮你？

**Reply by L-ZT** [reply]
https://bitbucket.org/chromiumembedded/cef/issues/1965/osx-ctrlkey-on-pinch

**Reply by L-ZT** [reply]
@tishion  `window.onwheel = e => console.log(e.altKey, e.ctrlKey)` 这行代码在CEF上始终返回`false false`，但是在浏览器上当按下对应键的时候能打印出`true`，

**Comment by tishion** [comment]
系统？
版本？
模式？

**Reply by L-ZT** [reply]
WIN10的系统，QT版本5.15.2，QCefView用的是最新版本，debug和release模式都存在这个问题

**Reply by L-ZT** [reply]
你好，请问这个问题有什么接口可以处理吗

**Reply by tishion** [reply]
@L-ZT 我在Qt 6.4版本上没有问题。

相关事件的处理在这里：
https://github.com/CefView/QCefView/blob/21cba754694ae814721d01d5cb83fe7b2736aad7/src/details/QCefViewPrivate.cpp#L832-L860

843行，修饰按键的状态是通过Qt的QWheelEvent直接取出来的，如果你在5.12.2的版本上无法正常工作，那可能就是Qt这个版本的QWheelEvent.modifiers有问题了。

**Reply by L-ZT** [reply]
好的，多谢
